_posts/2023-07-17-13.0.0-release.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Apache Arrow 13.0.0 Release"
+date: "2023-07-17 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 13.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 12.0.0 release, Marco Neumann, Gang Wu, Mehmet Ozan Kabak and Kevin Gurney
+have been invited to be committers.
+Matt Topol, Jie Wen, Ben Baumgold and Dewey Dunnington have joined the
+Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
```

Comment:
@zeroshade could you help with the Go notes for 13.0.0 ?

Reply:
will do

Reply:
```suggestion
## Go notes

### Enhancements

#### Arrow

* Compute arithmetic functions are now available for Float16 ([GH-35162](https://github.com/apache/arrow/issues/35162))
* Float16, Large* and Fixed types are all now supported by the CSV reader/writer ([GH-36105](https://github.com/apache/arrow/issues/36105) and [GH-36141](https://github.com/apache/arrow/issues/36141))
* CSV Reader uses `AppendValueFromString` for extension types and properly reads empty values as null ([GH-35188](https://github.com/apache/arrow/issues/35188) and [GH-35190](https://github.com/apache/arrow/issues/35190))
* [Substrait](https://github.com/substrait-io/substrait-go) expressions can now be executed using the Compute library ([GH-35652](https://github.com/apache/arrow/issues/35652))
* You can now read back values from Dictionary Builders before finishing the array ([GH-35711](https://github.com/apache/arrow/issues/35711))
* `MapType.ValueField` and `MapType.ValueType` are now deprecated in favor of `MapType.Elem().(*StructType)` ([GH-35909](https://github.com/apache/arrow/issues/35909))
* Multiple equality functions which have been deprecated since v9 have  now been removed (Such as `array.ArraySliceEqual` in favor of `array.SliceEqual`) ([GH-36198](https://github.com/apache/arrow/issues/36198))
* `ValueStr` method on Timestamp arrays now includes the zone in the output ([GH-36568](https://github.com/apache/arrow/issues/36568))
* *BREAKING CHANGE* `FixedSizeListBuilder.AppendNull` no longer requires manually appending nulls to the underlying list  ([GH-35482](https://github.com/apache/arrow/issues/35482))

#### Flight

* FlightSQL driver supports non-prepared queries now ([GH-35136](https://github.com/apache/arrow/issues/35136))

#### Parquet

* Error messages in row group writer have been improved ([GH-36319](https://github.com/apache/arrow/issues/36319))

### Bug Fixes

* Cross architecture build failures with v12.0.1 have been fixed ([GH-36052](https://github.com/apache/arrow/issues/36052))

#### Arrow

* It is now possible to build the Arrow Go lib using tinygo for building smaller WASM binaries ([GH-32832](https://github.com/apache/arrow/issues/32832))
* `Fields` method for Schema and StructType now returns a copy of the slice to ensure immutability ([GH-35306](https://github.com/apache/arrow/issues/35306) and [GH-35866](https://github.com/apache/arrow/issues/35866))
* `array.ApproxEqual` for Maps now allows entries for a given element to be presented in any order ([GH-35828](https://github.com/apache/arrow/issues/35828))
* Fix issues with decimal256 arrays ([GH-35911](https://github.com/apache/arrow/issues/35911), [GH-35965](https://github.com/apache/arrow/issues/35965), and [GH-35975](https://github.com/apache/arrow/issues/35975))
* StructType now allows duplicate field names correctly ([GH-36014](https://github.com/apache/arrow/issues/36014))

#### Flight

* Fix crash in client middleware ([GH-35240](https://github.com/apache/arrow/issues/35240))

### Parquet

* Various memory leaks addressed in pqarrow package ([GH-35015](https://github.com/apache/arrow/issues/35015))
* Fixed panic for `ListOf(types)` if null ([GH-35684](https://github.com/apache/arrow/issues/35684))

```